department-of-veterans-affairs/va.gov-cms
# VACMS-19496: Installation and enabling of content_sync module with settings, a README, and an exported piece of content.

## Description

Relates to #19496 

## Testing done
Local testing was creating the cloned content and exporting as yml file.

## Screenshots


## QA steps

What needs to be checked to prove this works?
Will need to test on tugboat to see if importing the content there works.
What needs to be checked to prove it didn't break any related things?
No change to any current look or behavior to site
What variations of circumstances (users, actions, values) need to be checked?

### Definition of Done


### Select Team for PR review
